app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -15,24 +15,58 @@ window.addEventListener('job-done', function(e){
 
 document.addEventListener('turbolinks:load', function() {
   if ($('body.upload.new').length) {
-    $('[data-behavior=type-select]').on('change', function() {
+    $('[data-behavior=type-select]').on('change', function(e) {
       var $nodeSelect = $('select option[value="node"]:selected').parent();
 
+      // Disable Node Label option
       $('[data-behavior=type-select]').each(function(i, select) {
```

Comment:
I know this isn't the correct place to review this but I just came across of the `.not` operator, and maybe we could change it use that and skip the loop altogether?

```javascript
      if ($nodeSelect.length) {
        $('[data-behavior=type-select]').not($nodeSelect).find('option[value="node"]').attr('disabled', 'disabled');
      } else {
        $('[data-behavior=type-select] option[value="node"]').removeAttr('disabled');
      }
```